jira-environment-ticket-create/action.yml

```diff
@@ -28,6 +31,10 @@ inputs:
   urlField:
     description: 'JSON field for the environment URL'
     required: true
+  jiraEnvironmentField:
+    description: 'JQL field used to find the given environment'
+    required: false
+    default: 'Environment Key[Short text]'
```

Comment:
Strange default?

Reply:
Yeah, whats stranger is it doesn't get overwritten :D
Took the piece of code for checking the ticket from jira-environment-ticket-delete and and keeping it the same here.